sentences/en/homeassistant_HassRespond.yaml

```diff
@@ -0,0 +1,23 @@
+language: "en"
+intents:
+  HassRespond:
+    data:
+      - sentences:
+          - "(hello|hi) home assistant"
```

Comment:
```suggestion
          - "(hello|hi)[ home assistant]"
```

Reply:
"hello" and "hi" by themselves are too short to get recognized until we fix the VAD issue :slightly_frowning_face:

Reply:
Just updated it from `(hello|hi)[ home assistant]` to `(hello|hi) [home assistant]`

Reply:
Didn't see Mike's comment but we defo need to get standalone "Hello" or "Hi" commands before production release. Raised on Discord